modules/i3s/src/lib/tileset/i3s-tileset.js

```diff
@@ -250,6 +240,14 @@ export default class I3STileset {
     this.stats.get(POINTS_COUNT).count = pointsRenderable;
   }
 
+  update(frameState) {
+    // TODO: only update when camera or culling volume from last update moves (could be render camera change or prefetch camera)
+    this._updatedVisibilityFrame++;
+    this._cache.reset();
+    this.options.onTraverseEnd = this._onTraverseEnd.bind(this);
```

Comment:
Nit: It seems strange that we bind the passed in callback, and also that we bind it on every update...